raco/algebra.py

```diff
@@ -462,6 +478,15 @@ def get_equijoin_condition(col0, col1):
         return expression.EQ(expression.UnnamedAttributeRef(col0),
                              expression.UnnamedAttributeRef(col1))
 
+    def partitioning(self):
+        """ The schemas are mutually exclusive so union the partition attributes"""
+        return RepresentationProperties(hash_partitioned=set.union(self.left.partitioning(),
+                                                                self.right.partition()))
+
+    def scheme(self):
+        """Return the scheme of the result."""
+        return self.left.scheme() + self.right.scheme()
```

Comment:
`SymmetricBinaryOperator` takes two identical schemes. Difference falls under this category. Maybe I'll refactor to `SymmetricSchemeBinaryOperator` to avoid confusion

`CompositeBinaryOperator` strictly concats two schemas.